eclipse-theia/theia
# repo: cleanup dependencies

#### What it does


The pull-request includes the following changes:
- [x] removes any unused dependencies (dependencies declared but not actually used).
- [x] adds any dependencies which are used but never declared in the respective `package.json`. 

#### How to test


- confirm that the repo builds, and the application starts normally
- confirm that the tests pass
- confirm that the removed dependencies aren't actually used for the given extension

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

Signed-off-by: vince-fugnitto <vincent.fugnitto@ericsson.com>
